templates/leaderboard/leaderboard.html

```diff
@@ -9,24 +9,76 @@
 {% endblock %}
 
 {% block content %}
-    <h2>Investments</h2>
-    <div class="break">
-        <hr>
-    </div>
     <div class="center">
-        {% include "leaderboard/includes/list.html" with name='Artists' leaders=top_raised %}
-        {% include "leaderboard/includes/list.html" with name='Investors' leaders=top_invested %}
-    </div>
-    <h2>Earnings</h2>
-    <div class="break">
-        <hr>
-    </div>
-    <div class="center">
-        {% include "leaderboard/includes/list.html" with name='Artists' leaders=top_earned_artists %}
-        {% include "leaderboard/includes/list.html" with name='Investors' leaders=top_earned_investors %}
+        {% include "leaderboard/includes/list.html" %}
```

Comment:
We can use this `{% includes %}` a little more intelligently since we are repeating the same code multiple times.

If you include `list.html` with `name='Investors'` and `canvas_id='InvestorChart'` in the first line and then `name=Artists` and `canvas_id=ArtistChart` in the second line, we can reuse the same HTML in `list.html`.

Once you do that, delete the artist heading and chart and then replace the "Investors" heading with `{{ name }}` and the "InvestorChart" id with `{{ canvas_id }}`.